01_Core/week02/submissions/index.html

```diff
@@ -0,0 +1,106 @@
+<!--
+VERY EASY: Create a bulleted list of the colors of the rainbow
+
+EASY: Create an HTML page with a main headline, two subheads in their own sections and supporting text for each subhead. Topic is up to you. Make sure you have a page title.
+
+MEDIUM: Using semantic HTML, add a footer with a copyright year and social media links (minimum of 3). Add a sidebar with links to outside content.
+
+HARD: Create a contact page with an HTML form with a GET method that will collect the following: First Name, Last Name, Gender (have them pick), Favorite sport (from a dropdown list of at least 5), Checkboxes for what you like on your pizza (from a list of at least 5) and
+the submit button should read, "Save".
+
+VERY HARD: On the same site, create a horizontal navigation of 5 items, one of the items should have a dropdown menu when you hover over it.
+
+VERY HARD: Given a comma separated string of full names, use JavaScript to create an alphabetical-by-Last-Name, ordered list and have it display in an HTML page on your site. Make the last names bold and red using CSS (NOT inline styles).
+
+## Note: Don't freak out about the last one
+
+LAST, BUT NOT LEAST: Make sure to post your site in Github pages. 
+-->
+
+<!-- To Do: 
+1) Make contact form
+2) The last one
+3) Add links to social media
+4) Add real content
+5) Clean up -->
+
+
+<!doctype HTML>
+
+<html>
+
+	<head>
+		<title>Testing, Testing, 1 2 3 </title>
+		<link rel="stylesheet" href="style.css">
+	</head>
+
+	<body>
+		<header>
+		<h1>Hello there</h1> 
+	</header>
+	
+	<div class="header clearfix">
+			<p>Nav 1</p>
```

Comment:
This looks like the start of a list of links, so semantically you should use an unordered list. You did this in the sidebar.

Reply:
I used an unordered list originally and struggled to get the dropdown div to show inline (at least based on the tutorials that I could find online). I will explore this more to figure out how to use a list.

Reply:
Most recent commit to this branch has the top navigation as an unordered list.